api/src/main/java/jakarta/servlet/ServletContext.java

```diff
@@ -212,6 +212,10 @@ public interface ServletContext {
      * application) security constraints. Care should be taken both when constructing the path (e.g. avoid unsanitized user
      * provided data) and when using the result not to create a security vulnerability in the application.
      * 
+     * <p>
+     * The provided {@code path} parameter is canonicalized as per section 3.5.2 of the specification before being used to
```

Comment:
I don't really like this. As far as I can see this would actually set a new precedent, as none of the other javadoc we have explicitly references a section of the spec (I could be wrong, but it seemed like all other references were to RFC's rather than the Servlet spec).

If we are going to do this I think we need to explicitly specify the version of the spec, as at some point the section number will change and we may forget to update the javadoc.

Reply:
Turns out I was wrong, there is this:

https://github.com/jakartaee/servlet/blob/100fd4e8d817ca2ca1f9e892c80e1c29b07e2aa0/api/src/main/java/jakarta/servlet/http/HttpServletRequest.java#L257

This does not even specify which spec, let alone the version :-)

I really think that over time this will likely become outdated and rot, as these number can change if additional items are added, and it would be very easy to miss that they need to be updated, although I don't really have a good solution.

Reply:
We should probably come up with a very standard javadoc style for referencing a specific spec document and the section within it, so that we can easily find and update in future.

Reply:
How about this (with hyperlink to the right section in the HTML version of the spec):
[Servlet 6.0, 3.5.2](https://jakarta.ee/specifications/servlet/6.0/jakarta-servlet-spec-6.0.html#uri-path-canonicalization)
We might need to tweak to working around the reference so that style works everywhere.

Reply:
That works for me

Reply:
Works for me as well.